hello-slack/app.py

```
@@ -0,0 +1,378 @@
+            try:
+                response = client.chat.completions.create(
+                    # model="gpt-4o-mini-2024-07-18",
+                    model="gpt-4o-mini",
+                    messages=messages
+                )
+                chatgpt_response = response.choices[0].message.content
+                logging.info(f"Received response from ChatGPT: {chatgpt_response}")
+                # スレッド内で返信するために thread_ts を指定
+                say(f"<@{user}> {chatgpt_response}", thread_ts=thread_ts)
+            except Exception as e:
+                logging.error(f"Error calling OpenAI API: {e}")
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)